HW1/P4/P4.py

```diff
@@ -0,0 +1,61 @@
+
+import pyspark
+from pyspark import SparkContext
+#from P4_bfs import BFS, iterate
+from P4_bfs2 import BFS, iterate
+import time
+
+sc = SparkContext()
+#f = open('graph.txt', 'w')
+chars = open('chars.txt', 'w')
+
+
+def char_to_edges(lstc):
```

Comment:
I like the way you implemented your function but I think we were suppose to do as much as we could via rdds rather than python structures. I think this is possible to do with just spark functions.